src/test/java/com/kainos/ea/integration/WebServiceIntegrationTest.java

```diff
@@ -30,4 +31,12 @@ void getEmployees_shouldReturnListOfEmployees() {
         assertTrue(response.size() > 0);
     }
 
+    @Test
+    void getSpecification_shouldReturnListOfSpecification() {
+        List<JobSpecification> response = APP.client().target("http://localhost:8080/api/job-specification")
+                .request()
+                .get(List.class);
+        assertTrue(response.size() > 0);
```

Comment:
can you also assert that the other values being returned are not null

Reply:
done some refactor here